client/my-sites/exporter/post-type-options.jsx

```diff
@@ -12,18 +12,22 @@ import FormRadio from 'components/forms/form-radio';
 import Select from './select';
 import Label from 'components/forms/form-label';
 
-import { setPostType } from 'state/site-settings/exporter/actions';
+import { setPostType, setPostTypeFilters } from 'state/site-settings/exporter/actions';
 import {
 	getPostTypeOptions,
+	getPostTypeValues,
 	getSelectedPostType,
 } from 'state/site-settings/exporter/selectors';
 
 const mapStateToProps = ( state, ownProps ) => {
 	const siteId = state.ui.selectedSiteId;
 	const options = getPostTypeOptions( state, siteId, ownProps.postType );
+	const values = getPostTypeValues( state, siteId, ownProps.postType );
```

Comment:
I've updated the naming convention a bit, to map basically to something like this:

![export-naming](https://cloud.githubusercontent.com/assets/416133/15565785/5300cecc-2360-11e6-82a4-260e65fd7890.png)

For props used by this component, I've changed these to `fields` and `fieldValues`, as adding 'postType' to the front might be redundant (since it's a post type specific component), ie:

```diff
-const options = getPostTypeOptions( state, siteId, ownProps.postType );
-const values = getPostTypeValues( state, siteId, ownProps.postType );

+const fields = getPostTypeFieldOptions( state, siteId, ownProps.postType );
+const fieldValues = getPostTypeFieldValues( state, siteId, ownProps.postType );
```